reporting-anything/src/constants.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const CHROMIUM_PATH = '../.chromium/headless_shell';
```

Comment:
can this be automatically downloaded?

Reply:
I can add a bootstrap script to add dependencies in package.json and chromium